microsoftgraph/microsoft-graph-explorer-v2
# Feat/code snippet

## Overview

Adds code snippets to Graph Explorer
### Demo

![image](https://user-images.githubusercontent.com/12011447/64448942-7d511780-d0e7-11e9-88f3-457821b87922.png)

### Notes

This PR uses Hooks. Learn more about hooks [here](https://reactjs.org/docs/hooks-intro.html)
## Testing Instructions

* Launch Graph Explorer
* Observe a Code Snippets tab
* Click on it
* Observe that snippets are loaded for the supported languages.
